en/docs/services/service-concepts.md

```diff
@@ -0,0 +1,113 @@
+# Concepts
+
+Before you start creating services with Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know.  
+ 
+## Service
+
+A service in Choreo is a logical representation of a repeatable activity that has a specified outcome.
+
+## Service Sample
+
+A service sample in Choreo is a ready-to-use service that you can easily use depending on your requirement. Choreo provides a comprehensive set of service samples that you can select and use depending on your requirement. 
+
+
+## Trigger
+
+A trigger in Choreo is an event or an action that can cause a Choreo application to start executing.
+
+Choreo supports the following trigger types:
+
+ - **API**: Allows you to expose an application as an API endpoint. 
+ - **Manual**: Allows you to trigger an application manually.
+ - **Schedule**: Allows to trigger an application based on a schedule that you set. You can specify the schedule in the form of a cron expression.
+   <html><div class="admonition note">
+     <p class="admonition-title">Note</p>
+     <p> A cron expression represents details of a schedule. It is a string that contains subfields separated by whitespaces. Each special character (*) represents seconds, minutes, hours, date, month, day, and year respectively.</p>
+     </div>
+     </html> 
+ - **Calendar**: Allows you to trigger an application based on events in a Google Calendar.
+ - **GitHub**: Allows you to trigger an application based on GitHub events and actions. The supported GitHub events and associated actions that can trigger a Choreo application are as follows:
```

Comment:
In the new UIs we don't have above thins in services view. They are in the integration view, So it seems if we put all these under services concepts it is confusing

Reply:
Removed content on the trigger concept as it is no longer applicable to services.